### PR TITLE
[prometheus-elasticsearch-exporter] Update pod labels

### PR DESCRIPTION
**Pull request by jimmybchopps**
#### What this PR does / why we need it

To increase traceability throughout the application by adding more of the standardized kubernetes labels to pod workloads in addition to the stateful set itself.

#### Special notes for your reviewer

Originally, the pod labels were only get the selector labels, this update adds all including the selector labels so no labels are being removed; only a few are being added to the pod.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by jimmybchopps** [comment]
PR ready for review please @svenmueller @desaintmartin @zeritti 

Please let me know if there are any questions.  Thanks!!!

File: charts/prometheus-elasticsearch-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 6.5.1
+version: 6.5.2
```
Comment (zeritti) [inline charts/prometheus-elasticsearch-exporter/Chart.yaml:4]: Please, raise the minor version number.
 ```suggestion
version: 6.6.0
```

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @jimmybchopps, for your PR. Please, see my comments below.

Adding the version label will cause an unnecessary restart of the exporter at release upgrade, even though the upgrade does not affect the pod template's spec.

**Review by zeritti — APPROVED** [review]
LGTM

**Pull request merged**